langchain4j-google-ai-gemini/src/main/java/dev/langchain4j/model/googleai/GeminiThinkingConfig.java

```diff
@@ -0,0 +1,71 @@
+package dev.langchain4j.model.googleai;
+
+
+/*
+ * @author : rabin
```

Comment:
We usually don't keep traces of `@author`.
Please remove that comment.
You'll be credited anyway in the Git log, and in the release notes too.